package.json

```diff
@@ -45,18 +45,22 @@
     "nyc": "^15.1.0",
     "sinon": "^9.0.3"
   },
-  "resolutions": {
-    "form-data": "^2.5.4",
-    "semver": "^7.0.0",
-    "string-width": "^4.0.0",
-    "wrap-ansi": "^7.0.0"
+  "pnpm": {
+    "overrides": {
+      "form-data": "^2.5.4",
+      "semver": "^7.0.0",
+      "string-width": "^4.0.0",
+      "wrap-ansi": "^7.0.0"
+    }
```

Comment:
These can be safely removed. Pnpm does a decent job while installing packages and will not hoist anything that could break the scripts.

In the past, our Git-based changelog complained due to issues with `require/import`. It's no longer an issue.

I was testing it on Node v22.19.

Reply:
`form-data` override is required to install safe version.

Reply:
Could we force installing `form-data@^4`?